std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
+  {
+    import std.file;
+    import std.path;
+    import std.algorithm;
+    import std.array;
+
+    return std.file.dirEntries(pathname, SpanMode.shallow)
+      .filter!((a) => a.isFile)
+      .map!((a) => std.path.baseName(a.name))
```

Comment:
Since this is documentation, I think we should use this style: `a => a.isFile`, without parentheses for the parameter list.